### PR TITLE
Add marker message

### PR DESCRIPTION
**Pull request by liorbond**

### CODE REVIEW

**Review by assafmo — APPROVED** [review]

File: packages/std/src/results/cosmos_msg.rs [hunk 1]
```diff
@@ -46,6 +46,7 @@ pub enum CosmosMsg<T = Empty> {
     Wasm(WasmMsg),
     #[cfg(feature = "stargate")]
     Gov(GovMsg),
+    LastMsgMark(T),
```
Comment (assafmo) [inline packages/std/src/results/cosmos_msg.rs:49]: Can we make it prettier, i.e. so devs won't need to pass a boilerplate `Empty {}` as `T`?

Comment (assafmo) [inline packages/std/src/results/cosmos_msg.rs:49]: Also, I think `FinilizeTx` is a better name here, as it's not just a "last message marker" within the tx, but also within that last message it's the last thing that will happen.

Comment (Cashmaney) [reply packages/std/src/results/cosmos_msg.rs:49]: I already merged the changes to the main PR, so I'll change this in a separate one

Comment (Cashmaney) [reply packages/std/src/results/cosmos_msg.rs:49]: I already merged the changes to the main PR, so I'll change this in a separate one

Comment (assafmo) [reply packages/std/src/results/cosmos_msg.rs:49]: That's alright, just asking out of curiosity. It's a matter of optimizing dev-x at this point.

**Pull request merged**